handlers.go

```diff
@@ -426,25 +426,34 @@ func (s *Server) handleAddIpfs(c echo.Context, u *User) error {
 		return err
 	}
 
+	filename := params.Name
+	if filename == "" {
+		filename = params.Root
+	}
+
```

Comment:
hrm.. as it stands, the `/content/add` API expects `collectionPath` to be the absolute path of the content you're uploading, including it's filename. I just did a bunch of parity fixes to make sure that is the case across both main node and shuttle in #99..

we should probably get on the same page about the API convention before merging, so these behave as expected

Reply:
> we should probably get on the same page about the API convention before merging, so these behave as expected

Absolutely! This bit of code is old though, I just replaced it a bit higher up because I needed `filename` to be set on a function call IIRC

Reply:
yeah, sorry -- I should've highlighted below where you append `filename` to the collection path